requirements.txt

```diff
@@ -1,2 +1,2 @@
-torch>=1.6
+torch>=1.7
```

Comment:
shy this is needed?

Reply:
Yes, torchvision 0.8 needs pytorch 1.7, so we need to bump :)

Reply:
ok, then make Retina as optional as we have with some others

Reply:
Ok we can make it optional, but why not update to PyTorch 1.7 ? It is out for 2 months and is stable. + CI didn't report any failure with PL 1.11 and torch 1.7 ?

Reply:
yes but we do not force users to upgrade their env unless it is critical for all package

Reply:
I'm not sure how to arrange this or refactor. @akihironitta @Borda   can you suggest changes / directly commit on top of this code ?

Reply:
I'm not sure how to arrange this or refactor. @akihironitta @Borda   can you suggest changes / directly commit on top of this code ?

Reply:
@oke-aditya By setting `torchvision>=0.8` in [requirements/models.txt](https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/requirements/models.txt), `pip install "pytorch-lightning-bolts[extra]"` will automatically install `torch>=1.7`, right? In that way, I think we can keep `torch>=1.6` as the basic requirement.

Reply:
So, shall we undo this change here and add `torchvision>=0.8` in requirements/models.txt?

Reply:
Done !